### PR TITLE
Rename eth to aleth in testing scripts

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

File: scripts/tests.sh [hunk 1]
```diff
@@ -70,39 +69,39 @@ then
...
-        ETH_PATH="eth"
+        ALETH_PATH="eth"
     else
         mkdir -p /tmp/test
         if grep -i trusty /etc/lsb-release >/dev/null 2>&1
         then
             # built from 5ac09111bd0b6518365fe956e1bdb97a2db82af1 at 2018-04-05
-            ETH_BINARY=eth_2018-04-05_trusty
-            ETH_HASH="1e5e178b005e5b51f9d347df4452875ba9b53cc6"
+            ALETH_BINARY=eth_2018-04-05_trusty
```
Comment (erak) [inline scripts/tests.sh]: Can we rename the binary as well?

File: scripts/tests.sh [hunk 2]
```diff
@@ -70,39 +69,39 @@ then
     fi
 fi
 
-function download_eth()
+function download_aleth()
 {
     if [[ "$OSTYPE" == "darwin"* ]]; then
-        ETH_PATH="$REPO_ROOT/eth"
+        ALETH_PATH="$REPO_ROOT/eth"
```
Comment (erak) [inline scripts/tests.sh]: Should this be also ``ALETH_PATH="$REPO_ROOT/aleth"``?

File: scripts/tests.sh (hunk 1 above)
Comment (axic) [reply scripts/tests.sh]: Yeah, that was in progress.

**Comment by axic** [comment]
This could be merged without updating the `aleth` binary first.

**Comment by chriseth** [comment]
Looks good but fails.

**Comment by axic** [comment]
Should be fixed now.

**Review by chriseth — DISMISSED** [review]

**Comment by axic** [comment]
I'm not sure yet while this fails.

**Comment by axic** [comment]
Rebased.

File: scripts/tests.sh [hunk 3]
```diff
@@ -70,58 +69,58 @@ then
...
-            ETH_BINARY=aleth_2018-06-20_artful
-            ETH_HASH="02e6c4b3d98299885e73f7db6c9e3fbe3d66d444"
+            ALETH_BINARY=aleth_2018-06-20_artful
+            ALETH_HASH="02e6c4b3d98299885e73f7db6c9e3fbe3d66d444"
         fi
-        wget -q -O /tmp/test/eth https://github.com/ethereum/cpp-ethereum/releases/download/solidityTester/$ETH_BINARY
-        test "$(shasum /tmp/test/eth)" = "$ETH_HASH  /tmp/test/eth"
+        ALETH_PATH="/tmp/test/aleth"
+        wget -q -O $ALETH_PATH https://github.com/ethereum/cpp-ethereum/releases/download/solidityTester/$ALETH_BINARY
+        test "$(shasum $ALETH_PATH)" = "$ALETH_HASH  $ALETH_PATH"
```
Comment (axic) [inline scripts/tests.sh:92]: There was a typo here previously the uppercase L was lowercase, hence the failures.

**Comment by axic** [comment]
I rebased this on develop and it seems the bytecodecompare script fails on appveyor. That is unrelated to this PR though.

@chriseth can we merge this?

**Review by chriseth — APPROVED** [review]

**Pull request merged**